src/components/inputcomponents/ImageInput/SingleInput.js

```diff
@@ -13,6 +16,22 @@ const singleInput = (props) => {
     updateImage(props.index, files[0]);
   }
 
+  function onSelect(files) {
```

Comment:
We already use Superagent for AJAX calls. Please import superagent here and make the call using that instead.